blockbuster/src/programs/bubblegum/mod.rs

```diff
@@ -215,6 +228,39 @@ impl ProgramParser for BubblegumParser {
     }
 }
 
+fn get_bubblegum_leaf_schema_event(app_data: Vec<u8>) -> Result<LeafSchemaEvent, BlockbusterError> {
+    let event_type_byte = if !app_data.is_empty() {
+        &app_data[0..1]
+    } else {
+        return Err(BlockbusterError::DeserializationError);
+    };
+
+    match BubblegumEventType::try_from_slice(event_type_byte)? {
+        BubblegumEventType::Uninitialized => Err(BlockbusterError::MissingBubblegumEventData),
+        BubblegumEventType::LeafSchemaEvent => Ok(LeafSchemaEvent::try_from_slice(&app_data)?),
+    }
+}
+
+// Convert from mpl-account-compression `ChangeLogEventV1` to
+// spl-account-compression `ChangeLogEventV1`.
+fn convert_mpl_to_spl_change_log_event(
```

Comment:
nit: Would this make sense as a From<> implementation to be more concise?

Reply:
Yeah I tried that, but it would have to be in spl-account-compression crate as either the trait or the type must be in your crate.